Update Spark/BigQuery documentation (#316)
mozilla/firefox-data-docs: src/cookbooks/bigquery.md

python -c 'from google.cloud import bigquery; print([d.dataset_id for d in bigquery.Client().list_datasets()])'
```

## From Spark

> We recommend [Storage API Connector](https://github.com/GoogleCloudPlatform/spark-bigquery-connector) for accessing

Reviewer: ```suggestion
We recommend the [Storage API Connector](https://github.com/GoogleCloudPlatform/spark-bigquery-connector) for accessing
```

BigQuery tables in Spark as it is the most modern and actively developed connector. It works well with the BigQuery
client library which is useful if you need to run arbitrary SQL queries (see example Databricks notebook) and load their

> results to Spark.

Reviewer: ```suggestion
results into Spark.
```

### On Databricks

> `shared_serverless_python3` cluster is configured with shared, default GCP credentials that will be automatically picked

Reviewer: ```suggestion
The `shared_serverless_python3` cluster is configured with shared default GCP credentials that will be automatically picked
```

> up by BigQuery client libraries. It also has the Storage API Connector library added - see example

Reviewer: ```suggestion
up by BigQuery client libraries. It also has the Storage API Connector library added as seen in the example
```
